src/Microsoft.SqlTools.ServiceLayer/Scripting/ScripterCore.cs

```diff
@@ -477,7 +477,21 @@ internal ScriptingScriptOperation InitScriptOperation(string objectName, string
 			    TargetDatabaseEngineEdition = GetTargetDatabaseEngineEdition(),
 			    TargetDatabaseEngineType = GetTargetDatabaseEngineType(),
 			    ScriptCompatibilityOption = GetScriptCompatibilityOption(),
-                IncludeIfNotExists = true
+                ScriptExtendedProperties = false,
```

Comment:
if you untabify this file then that would probably fix the misalignment issue here.